### ISSUE TITLE
Crashing

### ISSUE THREAD

**Issue by CaoTrongThang** [Oth3r/Sit#15]
### Describe your issue

Getting crash after pressing Singleplayer button

### Steps to reproduce

Create a world
Quit the world
Press Singleplayer button
-> Crash

### Put here any screenshots or videos (optional)

[crash-2024-12-23_11.51.32-client.txt](https://github.com/user-attachments/files/18225230/crash-2024-12-23_11.51.32-client.txt)

**Comment by Oth3r** [comment]
what did you do before hitting the singleplayer button?

**Comment by CaoTrongThang** [comment]
> what did you do before hitting the singleplayer button?

I created a world then quitted it, then I pressed the singleplayer button and got crashed

**Comment by Oth3r** [comment]
I couldn't recreate the crash, it might be a mod compatibility issue. Do you have any mods that mess with world loading?

**Comment by CaoTrongThang** [comment]
> I couldn't recreate the crash, it might be a mod compatibility issue. Do you have any mods that mess with world loading?

Yeah I think I do, I've like 300 mods, but it seems like I've to do the hard way to find out what mod cause it

**Comment by Oth3r** [comment]
please reopen if you find a way to reproduce, thank you!
